doc/examples/ex49/ex49.sh

```diff
@@ -26,7 +26,7 @@ gmt begin ex49
 	# WHy do we need the -R below? otherwise it fails to work
```

Comment:
Is there an answer to this question?

Reply:
Hm, let me look.

Reply:
I think the answer is this: Note that this is not a subplot but old-fashioned vertical shifting, yet it is modern mode.

We maintain history for two sorts of -R: plot  (RP) and data processing (RG).  Here, we use both and have already plotted a grid in line 16, so RP is set in the history.  Then we set RG by running the xyz2grd command. The next command is another grdimage call with a 12c upwards shift (so you and I know we are starting a new plot, probably).  However, without repeating the desired new plot region, GMT will search and find the current plot region, which is not what we want.  So the only way here is to tell GMT to explicitly use the new region.  Yes, a bit clumsy, but all because I did not make a subset out of this.

So perhaps I should rewrite this using subset then to avoid this dumb situation?

Reply:
Yes, using subplots would be better. Do you know if the RP and RG distinction is documented anywhere? If not, we should add it to https://docs.generic-mapping-tools.org/dev/cookbook/features.html#gmt-modern-mode-hierarchical-levels.

Reply:
I will look at the docs (this is under the hood stuff). Finally updated the scripts to use subplot and made a bunch of minor changes to simplify.